### PR TITLE
Modify leaseholder properties search

### PR DESCRIPTION
**Pull request by AnnaGolosova**
## Link to JIRA ticket

https://app.clickup.com/t/1xd0vny

### *What is the problem we're trying to solve*

As a User, on Calculate HOC screen, I would like to perform the Search

### *What changes have we introduced*

Modify leasehold-search endpoint to return charge totals

#### _Checklist_

- [ ] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [ ] Checked all code for possible refactoring
- [ ] Code pipeline builds correctly

### CODE REVIEW

File: FinanceServicesApi/V1/Infrastructure/EnumMapper.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+using FinanceServicesApi.V1.Boundary.Request.Enums;
+
+namespace FinanceServicesApi.V1.Infrastructure
+{
+    public static class EnumMapper
+    {
```
Comment (sandipchandra) [inline FinanceServicesApi/V1/Infrastructure/EnumMapper.cs:6]: We don't need this mapper, as the request will pass asset type as Dwelling

File: FinanceServicesApi/V1/Controllers/LeaseholdAssetSearchController.cs [hunk 2]
```diff
@@ -40,18 +35,21 @@ public LeaseholdAssetSearchController(IGetLeaseholdAssetsListUseCase getLeasehol
         public async Task<IActionResult> GetLeaseholdAssetsByAddress([FromQuery] LeaseholdAssetsRequest housingSearchRequest)
```
Comment (sandipchandra) [inline FinanceServicesApi/V1/Controllers/LeaseholdAssetSearchController.cs:35]: The request should contain the from year and we need to give response until the current year

File: FinanceServicesApi/V1/Boundary/Request/Enums/AssetType.cs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+namespace FinanceServicesApi.V1.Boundary.Request.Enums
+{
+    public enum AssetType
+    {
+        Property,
```
Comment (sandipchandra) [inline FinanceServicesApi/V1/Boundary/Request/Enums/AssetType.cs:5]: We don't need Asset Type as Property, as the request will pass asset type as Dwelling for Property.

**Review by sandipchandra — CHANGES_REQUESTED** [review]
Please review the comments

File: FinanceServicesApi/V1/Boundary/Request/Enums/AssetType.cs (hunk 3 above)
Comment (AnnaGolosova) [reply FinanceServicesApi/V1/Boundary/Request/Enums/AssetType.cs]: Done

File: FinanceServicesApi/V1/Controllers/LeaseholdAssetSearchController.cs (hunk 2 above)
Comment (AnnaGolosova) [reply FinanceServicesApi/V1/Controllers/LeaseholdAssetSearchController.cs:35]: Done

**Review by sandipchandra — APPROVED** [review]
approved

**Pull request merged**